lib/api/wallet_rpc.js

```diff
@@ -287,6 +290,68 @@ export class ArchethicRPCClient {
             )
     }
 
+    /**
+     * Signs many transactions.
+     * @param {Array<Object>} transactions where Object is a raw transaction. For example, a NFT creation transaction would look like this :
+     * ```
+     * {
+     *  "type": "token",
+     *  "version": 1,
+     *  "data": {
+     *  	"content": "{ \"name\": \"NFT 001\", \"supply\": 100000000, \"type\": \"non-fungible\", \"symbol\": \"NFT1\", \"aeip\": [2], \"properties\": {}}",
+     *  	"code": "",
+     *  	"ownerships":[],
+     *  	"ledger": {
+     *  		"uco": {
+     *  			"transfers": []
+     *  		},
+     *  		"token": {
+     *  			"transfers": []
+     *  		}
+     *  	},
+     *  	"recipients": []
+     *  }
+     * }
+     * ```
+     * @returns {Promise<Array<SignedTransaction>>}
+     */
+    async signTransactions(transactions) {
+        this._ensuresConnectionAlive();
+
+        return this.client?.request(
+            'signTransactions',
+            new RpcRequest(this.origin, transactions),
```

Comment:
The expected parameter if this function is not a single list of transaction, but an object with serviceName, pathSuffix and transactions :
```javascript
{
    serviceName: "toto",
    pathSuffix: "aeweb" //optional
    transaction: [tx1, tx2]
}
 ```